scripts/skills/perks/perk_rf_second_wind.nut

```diff
@@ -0,0 +1,32 @@
+this.perk_rf_second_wind <- ::inherit("scripts/skills/skill", {
+	m = {
+		ActionPointsTarget = 4	// set the current Action Points to this value if it's lower
+	},
+	function create()
+	{
+		this.m.ID = "perk.rf_second_wind";
+		this.m.Name = ::Const.Strings.PerkName.RF_SecondWind;
+		this.m.Description = ::Const.Strings.PerkDescription.RF_SecondWind;
+		this.m.Icon = "ui/perks/rf_second_wind.png";
+		this.m.Overlay = "rf_second_wind_effect";
+		this.m.Type = ::Const.SkillType.Perk;
+		this.m.Order = ::Const.SkillOrder.Perk - 5;
```

Comment:
- Why `-5`?
- On that note, a small difference like this doesn't really achieve anything in a predictable way because whenever a skill is added to an actor's `skill_container` via the `container.add` function it increases the `Order` of that skill object by `+1`. So, I'd say that this order should be `::Const.SkillOrder.Perk` or if you want it to trigger before all other perks (why?) then set it to an earlier perk order type declared in the vanilla SkillOrder table.

Reply:
I don't remember anymore. I will remove that modifier and we have to see if there was ever an issue. And next time I put a comment there explaining it.